src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -220,6 +220,47 @@ To see this, take a look at the job script for one of the downstream tasks:
    configuration.
 
 
+.. _Sequential Trigger Spawning:
+
+Sequential Trigger Spawning
+---------------------------
+
+Parent-less tasks (having no graph dependencies on other tasks) naturally spawn
+out to the runahead limit. The consequence for workflows with large runahead
+limits and/or task count may not only be UI clutter but also needless trigger
+checking. An example of this would be ``wall_clock``, where there's clearly
+no point in checking the future if the present trigger is unsatisfied.
+
+Sequential trigger spawning is a way of altering the default spawning behavior
+of these parent-less tasks. This is achieved by setting any number of triggers
+as sequential so that associated tasks only spawn their next (cycle point)
+instance on satisfaction of these triggers.
```

Comment:
You can use *sequential* xtriggers to avoid this problem: the next instance of a task (i.e., at the next cycle point) that depends on a sequential xtrigger will not be spawned until the previous xtrigger is satisfied. The ``wall_clock`` xtrigger is sequential by default.

Reply:
(Sorry for butchering your text - it's much easier to edit a paragraph than to write it in the first place 😁 )

Reply:
No problem, you're clearly more eloquent than I.